build.gradle

```diff
@@ -51,6 +51,16 @@ subprojects {
                 entry 'mapstruct-processor'
             }
 
+            // The AWS SDK v2 BOM is incorrectly pinning Cucumber to an
+            // old version, so we have to manually override the version
+            // of all Cucumber dependencies here.
+            dependencySet(group: 'info.cukes', version: '1.2.5') {
+                entry 'cucumber-core'
+                entry 'cucumber-java'
```

Comment:
do we need both java and java8?

Reply:
Yes, [`cucumber-java8` depends on `cucumber-java`](https://mvnrepository.com/artifact/info.cukes/cucumber-java8/1.2.5), and without pinning the version of `cucumber-java` we get errors because of version mismatches between the two.